airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1200,8 +1214,8 @@ def read_records(
             records = self.record_unnester.unnest(records)
 
             for record in records:
-                cursor = self._field_to_datetime(record[self.updated_at_field])
-                latest_cursor = max(cursor, latest_cursor) if latest_cursor else cursor
+                print(record)
```

Comment:
Left over from debugging?

Reply:
good catch! updating now...

Reply:
@aldogonzalez8 just wanted to let you know that I updated the PR to remove that print statement. Let me know if you spot anything else.

Reply:
Hey, sorry, need to fix HS CAT as one is broken because of missing data in test account before we can approve and merge, will let you know when is done.